.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: Branch Build
+
+on:
+  push:
+    branches:
+    - master
+    - version-*
+
+env:
+  GRADLE_OPTS: -Dorg.gradle.daemon=false -Xmx2g -Xms2g
+
+jobs:
+  branch-build:
+    # Only run this on repositories in the 'spinnaker' org, not on forks.
+    if: startsWith(github.repository, 'spinnaker/')
```

Comment:
Is it what we want?

Reply:
Yes, the workflow `build.yml` (which build on pushes to `master` and `release-` branches) is replaced by `operator-build.yml`, which builds on pushes to `gen-manifests` branch